### PR TITLE
Added ecm yaml files

### PR DESCRIPTION
**Pull request by jacarrete**
Added ecm yaml files

### CODE REVIEW

File: k8s/aat/common/ethos/ecm-consumer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ecm-consumer
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ethos/ecm-consumer:aat-eb96e9bc
+      ingressHost: ecm-consumer-aat.service.core-compute-aat.internal
```
Comment (timja) [inline k8s/aat/common/ethos/ecm-consumer.yaml:24]: this should be in the chart really and templated with the environment

Comment (jacarrete) [reply k8s/aat/common/ethos/ecm-consumer.yaml:24]: yeah it is in the chart, https://github.com/hmcts/ecm-consumer/blob/26fb1a5b18f557f81c4602fd2bd94dfcc8dc0402/charts/ecm-consumer/values.yaml#L4, i guess i can take that out from the aat and prod yaml right?

Comment (timja) [reply k8s/aat/common/ethos/ecm-consumer.yaml:24]: yes from all of them

Comment (jacarrete) [reply k8s/aat/common/ethos/ecm-consumer.yaml]: updated!

File: CODEOWNERS [hunk 2]
```diff
@@ -214,6 +214,7 @@ k8s/perftest/common/am/accessmgmt-api.yaml @hmcts/am-devs
 k8s/**/common/ethos/namespace.yaml @hmcts/ethos
 k8s/**/common/ethos/identity.yaml @hmcts/ethos
 k8s/**/common/ethos/repl-docmosis-service.yaml @hmcts/ethos
+k8s/**/common/ethos/ecm-consumer.yaml @hmcts/ethos
```
Comment (timja) [inline CODEOWNERS:217]: if it's not allowed in prod then this shouldn't be **

```suggestion
k8s/aat/common/ethos/ecm-consumer.yaml @hmcts/ethos
```

Comment (jacarrete) [reply CODEOWNERS:217]: I deleted the yaml file from prod. It should not be found anymore.

Comment (jacarrete) [reply CODEOWNERS:217]: > if it's not allowed in prod then this shouldn't be **

 deleted the yaml file from prod. It should not be found anymore.

**Pull request merged**